officemanager/department/tests/tests.py

```diff
@@ -1,3 +1,25 @@
 from django.test import TestCase
+from django.core.exceptions import ObjectDoesNotExist
+from department.models.models import *
```

Comment:
Please use from department.models import Departments, Employees instead of from department.models.models import *